lib/Component/VCard.php

```diff
@@ -357,7 +357,7 @@ function getValidationRules() {
 
             // FN is commented out, because it's already handled by the
             // validate function, which may also try to repair it.
-            // 'FN'           => '+',
+            // 'FN'           => '1',
```

Comment:
If I recall correctly, one use-case for multiple `FN` properties is different language variations. There's a `LANG` property as well for that purpose.